tests/nosetests/pyanaconda_tests/module_users_test.py

```diff
@@ -18,14 +18,13 @@
 # Red Hat Author(s): Martin Kolman <mkolman@redhat.com>
 #
 import unittest
-from unittest.mock import Mock, patch
+from unittest.mock import Mock
 
-from pyanaconda.modules.common.constants.interfaces import USER
 from pyanaconda.modules.common.constants.services import USERS
-from pyanaconda.modules.users.user import UserInterface, UserModule
 from pyanaconda.modules.users.users import UsersModule
 from pyanaconda.modules.users.users_interface import UsersInterface
-from tests.nosetests.pyanaconda_tests import check_kickstart_interface, check_dbus_property
+from pyanaconda.dbus.typing import get_variant, List, Str, Int, Bool
+from tests.nosetests.pyanaconda_tests import check_kickstart_interface
 
 
 class UsersInterfaceTestCase(unittest.TestCase):
```

Comment:
Please, check the coverage of the code to make sure that you didn't forget something:
```
pyanaconda/modules/users/installation.py                                        70     45     12      0    30%   40-44, 48, 51, 54, 70-72, 76, 79, 82-100, 113-115, 119, 122, 125-133, 146-148, 152, 155, 158-159
pyanaconda/modules/users/kickstart.py                                            5      0      0      0   100%
pyanaconda/modules/users/users.py                                              186     17     34      2    90%   129-130, 157-163, 168-171, 210-223, 286, 295, 156->157, 167->168
pyanaconda/modules/users/users_interface.py                                     84      2     14      0    98%   75, 86
```